fe1-web/model/objects/Hash.ts

```diff
@@ -5,6 +5,8 @@ import { Base64Data } from './Base64';
 export enum EventTags {
   MEETING = 'M',
   ROLL_CALL = 'R',
+  ELECTION = 'Election',
+  QUESTION = 'Question',
 }
```

Comment:
@pierluca did we agree on this? I think the point of having one byte as event tag is (in part) so that the hash doesn't leak any information rihgt?

Reply:
Update: actually we indeed agreed on that on a [merged PR](https://github.com/dedis/student_21_pop/pull/265/files). I think we should open an issue for that. What do you think?

Reply:
@RaulinN Maybe I'm missing something - can you explain how |tag| > 1 leaks information?